addons/sale/views/sale_portal_templates.xml

```diff
@@ -489,12 +489,14 @@
                         <tbody class="sale_tbody">
 
                             <t t-set="current_subtotal" t-value="0"/>
+                            <t t-set="current_subtotal_tax_inc" t-value="0"/>
                             <t t-set="lines_to_report" t-value="sale_order._get_order_lines_to_report()"/>
 
                             <t t-foreach="lines_to_report" t-as="line">
 
                                 <t t-set="current_subtotal" t-value="current_subtotal + line.price_subtotal"/>
-                                <t t-set="current_total" t-value="current_subtotal + line.price_total" t-if="sale_order.tax_calculation_rounding_method == 'round_per_line'"/>
+                                <t t-set="current_subtotal_tax_inc" t-value="current_subtotal_tax_inc + line.price_total"/>
+                                <t t-set="current_total" t-value="current_subtotal_tax_inc + line.price_total" t-if="sale_order.tax_calculation_rounding_method == 'round_per_line'"/>
```

Comment:
```suggestion
                                <t t-set="current_total" t-value="current_total + line.price_total" t-if="sale_order.tax_calculation_rounding_method == 'round_per_line'"/>
```

I guess this was previously wrong, because we probably don't want to add twice the line price to the total.